server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/CompactionManager.java

```diff
@@ -145,10 +145,11 @@ private long getDefaultThroughput(AccumuloConfiguration aconf) {
 
           HashMap<String,String> configsCopy = new HashMap<>(configs);
 
-          Map<String,String> defaultServiceConfigs =
-              Map.of(defaultServicePrefix + "planner", DefaultCompactionPlanner.class.getName(),
+          Map<String,
+              String> defaultServiceConfigs = Map.of(defaultServicePrefix + "planner",
+                  DefaultCompactionPlanner.class.getName(),
                   defaultServicePrefix + "planner.opts.executors",
-                  "[{'name':'deprecated','numThreads':" + numThreads + "}]");
+                  "[{'name':'deprecated', 'type':'internal', 'numThreads':" + numThreads + "}]");
```

Comment:
Rather than make this change, I think we should default to inferring an internal type when it is unspecified. Explicitly specifying `internal` should not be required.

Reply:
Implemented in 58673f8